Xamarin.Forms.Platform.UAP/SwitchRenderer.cs

```diff
@@ -40,10 +47,18 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			{
 				UpdateFlowDirection();
 			}
+			else if (e.PropertyName == Switch.OnColorProperty.PropertyName)
+				UpdateOnColor();
 		}
 
 		protected override bool PreventGestureBubbling { get; set; } = true;
 
+		void OnControlLoaded(object sender, RoutedEventArgs e)
```

Comment:
Hi guys, just wondering if somebody can help me to figure out what am I doing wrong getting this exception after update?

![image](https://user-images.githubusercontent.com/18675847/54688476-9227f780-4b15-11e9-8719-5b5862d899d4.png)

Reply:
It seems that `Control.GetChildren<Windows.UI.Xaml.Controls.Grid>().FirstOrDefault()` returns `null` for some reason.

Reply:
@paymicro thanks for prompt reply! This is a piece of xaml I'm using (first Switch is used to activate the second)
            < StackLayout Orientation="Horizontal" >
                < Switch x:Name="FirstSwitch" IsToggled="{Binding **FirstToggled**}" / >
            < /StackLayout >
            < StackLayout Orientation="Horizontal" IsVisible="{Binding **FirstToggled**}" >
                < Switch x:Name="SecondSwitch" IsToggled="{Binding SecondToggled}" / >
            < /StackLayout >

It seems if I use the same variable, used as IsToggled binding, somewhere else, in this case in the IsVisible binding, it breaks...
Have just created brand new XF App1, updated from nuget to the latest version from yesterday, added same xaml and it crashes.
Actually only rolling back to 3.5.0.169047 works for me, for the two 3.6 in nuget it seems to crash for such cases.